recipes/mlpy/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "3.5.0" %}
+{% set sha256 = "344fa75fbf9f76af72f6a346d5309613defc4d244bac13c218e509a51d68bf6a" %}
+#source file is named download, has to be unpacked manually, update build sh accordingly when updating version
+
+package:
+  name: mlpy
+  version: {{ version }}
+
+source:
+  url: https://sourceforge.net/projects/mlpy/files/mlpy%203.5.0/mlpy-{{ version }}.tar.gz/download
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+#  skip: True # [py27]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python >=2.6
+    - setuptools
+    - numpy =1.5
+    - scipy =0.11
+    - gsl =1.16
+  run:
+    - python
+    - numpy
+    - scipy
+    - gsl
+
+test:
+  importss:
+    - mlpy
+
+about:
+  home: http://mlpy.sourceforge.net/index.html
+  license: GPL3
+  summary: mlpy is a Python module for Machine Learning built on top of NumPy/SciPy and the GNU Scientific Libraries.
```

Comment:
Would you like to add yourself as maintainer? We appreciate if people "adopt" packages, so that if there is an issue or an update to the package, there is someone we can ping who's already familiar with the software.

Reply:
I just built this as requirement for squigglekit, never used it myself, but if nobody else is willing I can add myself

Reply:
Yeah, that's the way it usually goes. You end up being maintainer for most of the dependency tree of the things you want on here. I've even got X11 stuff over on CF on my list for that reason.

Reply:
Well, one for team it is then ;)